### PR TITLE
Week 3 tutorials

### PR DESCRIPTION
**Pull request by andreeanica16**
Follow the steps specified in the week 3 tutorial in order to learn how to make a comments section on my page.

### CODE REVIEW

**Comment by andreeanica16** [comment]
@iht

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
```
Comment (iht) [inline portfolio/src/main/java/com/google/sps/data/Comment.java]: 2020?

File: portfolio/src/main/java/com/google/sps/data/DataSent.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
```
Comment (iht) [inline portfolio/src/main/java/com/google/sps/data/DataSent.java]: 2020?

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
```
Comment (iht) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java]: 2020

**Review by iht — APPROVED** [review]
LGTM

Please check the year of the copyright messages.

**Pull request merged**